easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb

```diff
@@ -0,0 +1,63 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'Java'
```

Comment:
@edmondac Maybe it makes sense to use `OpenJDK` as software name here, to clearly discriminate from Oracle Java? That would complicate using this in the `Java/1.8` wrapper though (since `ModuleRC` requires that the names of the wrapper and the dependency match).